src/SpectatorView.Unity/Assets/SpatialAlignment/README.md

```diff
@@ -10,25 +14,77 @@ Not all spatial alignment strategies support all platforms. See the chart below
 | QR Code Detection      | x          |            | x       | x   |
 | ArUco Marker Detection |            | x          | x       | x   |
 
-## Application Flow
+## Key Concepts
+
+Before diving into the abstractions, we operate on two concepts when speaking of localization:
+
+- **Coordinate Space:** When a rotation/position is meant to be relative to a specific coordinate (location in the real world), we say it is in coordiante space. These roations/positions can be sent across the network and understood by each device that understand what coordiante they are associated with.
+- **Application World Space:** The rotations/positions that are set in Unity to `.position` and `.rotation` properties of `Transform` objets, are specific to the applications own "world space". This "world space" is used by the application to determine how to lay out content relative to each other; and this "world space" itself is relative to the positon/rotation of device when the application was launched.
+
+The following constructs compose the abstraction and facilitate the the localization processes:
+
+- **`ISpatialCoordinate`:** The abstract cosntruct symbolizing a world coordinate that can be used to convert between apllication's world space and coordinate-relative space.
+- **`ISpatialCoordinateService`:** A service for discovering and managing `ISpatialCoordiantes` based on a specific implementation.
+- **`SpatialLocalizerInitializer`:** The construct that begins and facilitates the creation/sharing of ISpatialCoordinates between the local and remote `SpatialLocalizer`.
+  - **`SpatialLocalizer`:** This related construct understands how to localize upon or create a `ISpatialCoordinate` for localization.
+- **`SpatialCoordinateSystemManager`:** The singleton manager that manages the incoming/outgoing networking connections, their associated localization state and assigned `ISpatialCoordinates` to them.
```

Comment:
subject nit: this may more easily read as "The singleton manager that monitors incoming/outgoing network connection, said connections associated localization states, and said connections assigned ISpatialCoordinates."

Reply:
I still think this one reads in a confusing manner. their can be singular which suggested the associated localization state is specific to the manager compared to the connection.